QRHunter/app/src/main/java/com/example/qrhunter/ui/qrcode/QrCodeViewModel.java

```diff
@@ -10,22 +14,16 @@
 import com.example.qrhunter.data.repository.QRCodeRepository;
 
 import java.util.ArrayList;
-/**
- * The view model for the QR code screen. This class updates itself when data changes.
- */
+import java.util.List;
+import java.util.Locale;
+
 public class QrCodeViewModel extends ViewModel {
     private final MutableLiveData<QRCode> qrCode = new MutableLiveData<>(null);
     private final MutableLiveData<Integer> scannedBy = new MutableLiveData<>(0);
     private final MutableLiveData<ArrayList<Comment>> comments = new MutableLiveData<>(new ArrayList<>());
     private QRCodeRepository qrCodeRepository = new QRCodeRepository();
     private CommentRepository commentRepository = new CommentRepository();
 
-    /**
-     * Gets the QR code with the specified ID.
-     *
-     * @param qrCodeId The ID of the QR code to get.
-     * @return The QR code to be returned.
-     */
```

Comment:
Curious why these comments are removed. Did u rewrite them somewhere else?

Reply:
oh i'll add them back i think i accidentally over wrote it during merging